common/lib/xmodule/xmodule/js/spec/video/video_full_screen_spec.js

```diff
@@ -30,8 +30,6 @@
                 var fullScreenControl = $('.add-fullscreen');
 
                 expect(fullScreenControl).toHaveAttrs({
-                    'role': 'button',
```

Comment:
Because we're using `<button>` as the element and specifying the role of the same is redundant :)